sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -76,4 +78,32 @@ export class ApplicationService {
   ): Promise<ProgramYearOfApplicationDto> {
     return ApiClient.Application.getProgramYearOfApplication(applicationId);
   }
+
+  /**
+   * Get application detail of given application
+   * @param applicationId
+   * @param userId
+   * @returns GetApplicationBaseDTO
+   */
+  async getApplicationDetail(
+    applicationId: number,
+    studentId: number,
+  ): Promise<GetApplicationBaseDTO> {
+    return ApiClient.Application.getApplicationDetails(
+      applicationId,
+      studentId,
+    );
+  }
+
+  /**
+   * Get application detail of given application
+   * @param applicationId
+   * @param userId
+   * @returns GetApplicationBaseDTO
```

Comment:
The returning object in the comment is different from the actually returned.

Reply:
comment updated